lib/compiler/passes/report-infinite-recursion.js

```diff
@@ -39,7 +39,7 @@ function reportInfiniteRecursion(ast) {
         throw new GrammarError(
           "Possible infinite loop when parsing (left recursion: "
             + visitedRules.join(" -> ")
-            + ").",
```

Comment:
I've also changed the error texts to be consistent in punctuation. I flipped a coin and got "no period at the end"

Reply:
Agreed. That points looks alien